extensions/amp-delight-player/0.1/amp-delight-player.js

```diff
@@ -25,11 +32,12 @@ import {removeElement} from '../../../src/dom';
 import {setStyle} from '../../../src/style';
 import {userAssert} from '../../../src/log';
 
-import {CSS} from '../../../build/amp-delight-player-0.1.css';
-
 /** @const */
 const TAG = 'amp-delight-player';
 
+/** @private @const */
+const ANALYTICS_EVENT_TYPE_PREFIX = 'video-custom-';
```

Comment:
Can you add a TODO to export this from a lower level, like `src`? (No need to do for this PR, I'll take care of it)